src/en/server-hosting/popular-forks-cdns.md

```diff
@@ -0,0 +1,39 @@
+# List of codebases with prebuilt server packages
+
+This document will serve as a list of forks known to have a prebuilt server software for players to download.
+
+This doc was last updated at 2024-10-01 13:52:04 UTC
+
+| Fork name | URL | Available platforms |
+|---|---|---|
+| Wizard's Den | [Link](https://wizards.cdn.spacestation14.com/fork/wizards) | Windows, macOS (x64/Intel), Linux (x64/ARM64) |
+| Harmony 14 | [Link](http://cdn.harmony14.com/fork/harmony/) | Windows, macOS (x64/Intel), Linux (x64/ARM64) |
+| Solaris 14 | [Link](https://ss14-cdn.mw-gc.com/fork/MWGSolaris14) | Windows, macOS (x64/Intel), Linux (x64/ARM64) |
+| Axolotl MRP | [Link](https://axolotl.yuniiworks.de/cdn/fork/axolotl/) | Linux (x64) |
+
+FAQ:
+
+- The fork I want is not listed here.
+	- You will need to contact the fork you want a server build for and ask them if they provide one. If they do suggest them to add themselves here.
+	
+- The fork I want is listed here, but my platform is not available for download.
+	- You will need to contact the fork asking if they are willing to build for your platform.
+	
+- The fork does not provide prebuilds (or any other issue where you are unable to retrive a server prebuild)
+	- Your only choice is to compile the forks codebase yourself if available. Instructions can be found [here](../general-development/setup/server-hosting-tutorial.md#level-2-server-with-custom-code)
+
+- I'm a server host and I want to add myself here.
+	- You are free to make a PR yourself in the docs or ping @vasilisiscool (322708417540259841) in #hosting on discord with all the information needed.
+
+<!-- 
+Rules for listing:
+1. Must have a decent playerbase on the hub and been on for at least a week.
+2. Must be a simple link to the download source, no discord invites or anything like that.
+3. Must have at least one server platform.
+4. No protected forks (Require a password for access to server builds)
+5. 18+/ERP forks must be expliticly stated in the fork name.
+6. Dehubbed servers or any servers breaking these rules will be removed.
+
+Template
+| SERVER NAME | [Link](SERVER_LINK) | Windows, macOS (x64/Intel), Linux (x64/ARM64) | 
```

Comment:
Could there be a column for the software license(s) that the fork is under? I think that's pretty important to know if you're hosting, considering some forks use AGPL.

Reply:
In most cases, users who download prebuilds are only interested in a sandbox environment. But i will add it